### DISCUSSION TITLE
💡 Improving Daily.dev: Engagement, Notifications, Links, Confirmation, Leaderboard, Reputation

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [dailydotdev/daily#860] [Feature Requests]
- The current algorithm registers a post as "read" if I click on it, regardless of how much time I spend on it. However, a better approach would be to require users to spend at least 10% of the estimated reading time, or to bookmark the post and revisit it later. This would ensure that users have genuinely engaged with the content, and also address the issue of users who only visit a post briefly and don't want tag to count towards their profile.

-  It would be great to have a feature that sends notifications for upvotes that contribute to our reputation, while ignoring others, except for the 50, 100, and other such significant upvotes.

- I believe that community engagement is the most effective way to promote the platform. We could add the daily.dev badge to our profiles on platforms such as GitHub, as discussed in this issue: https://github.com/simple-icons/simple-icons/issues/7402. Moreover, if the icons are added, we could also use other platforms like https://linkfree.eddiehub.io/icons to boost our visibility.

- When reporting a post from an Android device, there is no confirmation popup, which can be confusing and make it unclear whether the report was successfully submitted.

- Since daily.dev is a platform for discovering articles, it would be helpful to include links to our own articles on supported platforms like dev.to, and Medium, with appropriate branding similar to GitHub. This would help users discover our content and increase our visibility on these platforms.

- It would be great to have a weekly leaderboard of the most active community members, based on various factors including time spent on the app, comments, upvotes, and reputations gained, with active days given different levels of priority.

- It would be useful to include specific stats for reputation on the devcard, which would catch the attention of other users and give us more credibility.

- There have been no updates regarding my reported posts, and I am keen to know if they were removed correctly or not. Having this information will enable me to file more accurate reports in the future.

**Comment by idoshamun** [comment]
Hi @Anmol-Baranwal,

Thanks for continuously sharing your feedback with us. This is very helpful 🙏
Let me forward it internally, as this is quite a lot to process 😬

As for the reports, you will get notified only when we approve your report in case the post wasn't banned before.
I hope this makes sense to you.

**Reply by Anmol-Baranwal** [reply]
Thank you for letting me know about the report process and for being open to my suggestions. 
It means a lot and I really appreciate it.
